toml.md

```diff
@@ -259,12 +259,13 @@ String
 ------
 
 There are four ways to express strings: basic, multi-line basic, literal, and
-multi-line literal. All strings must contain only valid UTF-8 characters.
+multi-line literal. Strings can contain any valid Unicode codepoint except the
+following control characters: U+0000 to U+0008, U+000A to U+001F, and
+U+007F. Note that tab (U+0009) is allowed. Multi-line strings can also contain
+newlines (U+000A) and carriage returns (U+000D).
```

Comment:
I think that saying first that `U+000A` and `U+000D` are not allowed first (they are included in the ranges) and then adding an exception for Multi-line strings is kind of a double negative...

My opinion is that it would be better to not include these characters in the range and then specifically say that "basic" and "literal" (single-line) also don't allow them.

For example:

```markdown
multi-line literal. Strings can contain any valid Unicode codepoint except the
following control characters: U+0000 to U+0008, U+000B, U+000C, U+000E, U+001F, and
U+007F. Note that tab (U+0009) is allowed. Newlines (U+000A) and carriage returns (U+000D)
characters are forbidden in basic and literal string (non multi-line).
```